app/backend/lib/gis-upload.ts

```diff
@@ -71,7 +71,9 @@ gisUpload.post('/api/analyst/gis', limiter, async (req, res) => {
 
   const filename = Object.keys(files)[0];
   const uploaded = files[filename];
-
+  if (!!uploaded === false) {
```

Comment:
Couldn't this be simplified with just doing: 
```suggestion
  if (!uploaded) {
```

since it'll still implicitly convert anything falsey to a true.